src/App.js

```diff
@@ -24,20 +26,22 @@ class App extends Component {
   componentDidMount = async() => {
     const data = await fetchData(this.state.query, this.state.direction);
     this.setState({data})
+    this.setState({loading:false})
```

Comment:
You can actually join these two setState functions so that you're only calling it once. It would look like this: 

`this.setState({ data, loading: false})`